src/Storages/Statistics/TDigestStatistic.h

```diff
@@ -1,13 +1,16 @@
 #pragma once
 
 #include <Storages/Statistics/Statistics.h>
+#include <AggregateFunctions/QuantileTDigest.h>
 
 namespace DB
 {
 
+
 /// TDigestStatistic is a kind of histogram.
 class TDigestStatistic : public IStatistic
 {
+    friend class ColumnStatistics;
```

Comment:
Unless really really needed, it would be nicer if external classes could access only the public interface of this class (otherwise class members probably have the wrong visibility).